### PR TITLE
Add logs for check connection and connection schema view

### LINKED ISSUE #960: Trim Spaces in the UI forms inputs
## Tell us about the problem you're trying to solve
When copy/pasting info into the forms to setup a source or destination, i often end up selecting some unwanted trailing white-spaces at the end or at the beginning of what i am inputting

## Describe the solution you’d like
It'd be nice if the UI could automatically trim the spaces before and after the text fields before building the JSON configs.

## Describe the alternative you’ve considered or used
Otherwise we would have to add these trimming/check everywhere in each connector code as well

### LINKED ISSUE #1121: Add Logs when discover or check connection fail.
Please do this one after https://github.com/airbytehq/airbyte/issues/1118. We would like to user the new log view for this.

## Tell us about the problem you're trying to solve
When discover_schema or check connection fail, show the logs view (the same one we show in the job history for sync) at the bottom of the page. The logs should be accessible via `/v1/jobs/list`

ids structures use to call that endpoint
check connection for a source `configType: check_connection_source`, `configId: <source id>`
check connection for a destination `configType: check_connection_destination`, `configId: <destination id>`
discover schema `configType: discover_schema`, `configId: <source id>`

### LINKED ISSUE #1417: Replace the word incomplete in job history
## Tell us about the problem you're trying to solve
* The word "incomplete" is confusing to users. When a job status is "incomplete", in the UI we should display the word "retrying" instead of "incomplete"

### PR DESCRIPTION
**Pull request by jamakase**
## What
Add block with `Failed Logs` (reused component from `job history`) when `check_connection` or `discover_schema` requests failed.

Closes #1121
Closes #1417 
Closes #960

### CODE REVIEW

**Comment by Jamakase** [comment]
**Examples of Logs view**

close logs
open logs

File: airbyte-webapp/src/pages/OnboardingPage/components/useGetStepsConfig.tsx [hunk 1]
```diff
@@ -39,29 +39,32 @@ const UseGetStepsConfig = (
     () => [
       {
         id: StepsTypes.CREATE_SOURCE,
-        name: <FormattedMessage id="onboarding.createSource" />,
-        onSelect: hasSources
-          ? () => updateStep(StepsTypes.CREATE_SOURCE)
-          : undefined
+        name: <FormattedMessage id="onboarding.createSource" />
+        // don't navigate by steps for now
```
Comment (cgardens) [inline airbyte-webapp/src/pages/OnboardingPage/components/useGetStepsConfig.tsx:43]: i'm fine with this. why did you choose this? just curious. it seems like this feature has been a little finicky.

**Review by cgardens — APPROVED** [review]

**Comment by cgardens** [comment]
@Jamakase thank you for the screenshots! it looks great.

File: airbyte-webapp/src/pages/OnboardingPage/components/useGetStepsConfig.tsx (hunk 1 above)
Comment (Jamakase) [reply airbyte-webapp/src/pages/OnboardingPage/components/useGetStepsConfig.tsx:43]: Yeah, we discussed it during the call, that we need to disable those button. Currently this functionality is a bit broken, should we open 1 more task to fix it? As we have changed check connection functionality - it is now much more easier to support it.

**Comment by michel-tricot** [comment]
@Jamakase Can I merge or are you still pushing to that branch?

**Pull request merged**